src/status_im/ui/screens/chat/sheets.cljs

```diff
@@ -16,6 +16,7 @@
   (re-frame/dispatch event))
 
 (defn one-to-one-chat-accents [chat-id]
+
```

Comment:
Unneeded new line

Reply:
Ignore old ui

Reply:
If a PR has changes to a file it cannot be ignored